irohad/consensus/yac/yac_gate.hpp

```diff
@@ -29,7 +29,9 @@ namespace iroha {
       class YacGate : public network::ConsensusGate {};
 
       /**
-       * Provide gate for ya consensus
+       * Provide gate for YAC
+       * //TODO: I don't really like this class name or file name. The key point here is not about creating hashes
```

Comment:
The idea was that Consensus gate was an interface to vote for block, and to receive committed blocks, and this is internal YAC gate which votes only for hashes, not complete blocks, because YAC does not have knowledge of complete blocks with transactions. Could you suggest another name?